src/screencast/screencast.c

```diff
@@ -25,6 +25,7 @@ void xdpw_screencast_instance_init(struct xdpw_screencast_context *ctx,
 	cast->framerate = out->framerate;
 	cast->with_cursor = with_cursor;
 	cast->refcount = 1;
+	cast->type = XDPW_INSTANCE_SCP_SHM;
```

Comment:
Currently this declaration is used to change between shm screencopy and dmabuf screencopy.